pkg/server/biz/scm/bitbucket/bitbucket-server/prs.go

```diff
@@ -0,0 +1,90 @@
+/*
+Copyright 2017 caicloud authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package server
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+)
+
+// PullRequest represents a BitBucket Server pull request on a repository.
```

Comment:
The file name `prs.go` is not readable, how about `pull_requests.go`

Reply:
Agree